config.yaml

```diff
@@ -76,6 +76,7 @@ orgs:
     - mabegglen
     - th-basler
     - MartinV-DevOps
+    - abisek9186
```

Comment:
As stated in our guidelines we require the Baloise eMail identity to be used: https://github.com/baloise-incubator/org/commit/5c663eaa71030722422995eeef83375cc647e753.patch please amend your commit with the correct author Identity.